src/frontend/src/utils/receipt.test.tsx

```diff
@@ -33,11 +33,11 @@ describe('utils > receipt', () => {
   } as IParty;
 
   const language: ILanguage = {
-    receipt_platform: {
+    receipt: {
       date_sent: 'Dato sendt',
       sender: 'Avsender',
       receiver: 'Mottaker',
-      reference_number: 'Referansenummer',
+      ref_num: 'Referansenummer',
     },
   };
```

Comment:
This was changed from `receipt_platform` to `receipt` now. If this doesn't make the test fail, it probably isn't needed at all.